lib/autoproj/ops/configuration.rb

```diff
@@ -227,17 +242,30 @@ def load_and_update_package_sets(root_pkg_set, only_local = false)
                 end
                 by_repository_id[repository_id] = [vcs, imported_from]
 
+                # Make sure the package set has been already checked out to
+                # retrieve the actual name of the package set
                 if !vcs.local?
                     update_remote_package_set(vcs, only_local)
-                    create_remote_set_user_dir(vcs)
+                    raw_local_dir = PackageSet.raw_local_dir_of(vcs)
+                    required_remotes_dirs << raw_local_dir
                 end
-
                 name = PackageSet.name_of(manifest, vcs)
+
+                names = by_name.collect { |k,v| k }
+                Autoproj.debug "Trying to load package_set: #{name} from definition #{repository_id}"
+                Autoproj.debug "Already loaded package_sets are: #{names}"
+
                 if already_loaded = by_name[name]
                     already_loaded_pkg_set, already_loaded_vcs = *already_loaded
                     if already_loaded_vcs != vcs
                         if imported_from
-                            Autoproj.warn "#{imported_from.name} auto-imports a package set from #{vcs}, but a package set with the same name (#{name}) has already been imported from #{already_loaded_vcs}, I am skipping this one"
+                            Autoproj.warn "redundant auto-import by #{imported_from.name} for package set '#{name}'."
+                            Autoproj.warn "    A package set with the same name (#{name}) has already been imported from"
+                            Autoproj.warn "        #{already_loaded_vcs}"
+                            Autoproj.warn "        with server configuration '#{repository_configuration_label(already_loaded_vcs)}'."
```

Comment:
I can remove the server configuration information, but I would not want to invest much more time on that topic!? 

Current message see below. 
``` ruby
  WARN: redundant auto-import by main configuration for package set 'rock.dfki'.
  WARN:     A package set with the same name (rock.dfki) has already been imported from
  WARN:         git:git@git.hb.dfki.de:/rock-dfki/package_set.git push_to=git@git.hb.dfki.de:/rock-dfki/package_set.git repository_id=dfkigit:/rock-dfki/package_set.git retry_count=10
  WARN:         with server configuration 'dfkigit'.
  WARN:     Skipping the following one: 
  WARN:         git:git@git.hb.dfki.de:/rock-dfki/package_set.git push_to=git@git.hb.dfki.de:/rock-dfki/package_set.git repository_id=spacegit:/rock-dfki/package_set.git retry_count=10
  WARN:         with server configuration 'spacegit'.